src/main/java/com/zombiecastlerush/building/Room.java

```diff
@@ -37,18 +38,26 @@ public void setDescription(String description) {
         return connectedRooms;
     }
 
-    public Inventory getInventory() {
-        return inventory;
+    public Challenge getChallenge() {
+        return challenge;
+    }
+
+    public void setChallenge(Challenge challenge) {
```

Comment:
Is it necessary to `setChallenge` here? Why not initialize it in the instance variables?

Reply:
will keep set Challenge